Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,34 @@
-﻿namespace Logger;
+﻿using System;
+
+namespace Logger;
 
 public static class BaseLoggerMixins
 {
-    
+    public static void Error(this BaseLogger logger, string message, params object[] args)
+    {
+        if (logger == null)
```

Comment:
```suggestion
        if (logger is null)
```
Reminder to self. I should use is null instead.